Library/Homebrew/download_strategy.rb

```diff
@@ -388,6 +388,25 @@ def _fetch
   end
 end
 
+class CurlSourceForgeSecureMirrorDownloadStrategy < CurlDownloadStrategy
+  def fetch
+    if !ENV["HOMEBREW_NO_INSECURE_REDIRECT"].nil?
+      oldurl = @url
+      @url = https_mirror
+      ohai "Sourceforge HTTPS: #{oldurl}"
+      ohai "=> converted to #{@url}"
```

Comment:
Oh, that was not very clear in colored view
![screen shot 2015-08-14 at 23 33 36](https://cloud.githubusercontent.com/assets/384824/9284546/fd162d04-42dc-11e5-8c51-5ea51bf77b73.png)

Also the message is not very clear about what has been done and why